components/logstash/scripts/create.py

```diff
@@ -22,53 +22,57 @@
 ctx_properties = utils.ctx_factory.create(LOGSTASH_SERVICE_NAME)
 
 
-def install_logstash():
-
-    logstash_unit_override = '/etc/systemd/system/logstash.service.d'
-
-    logstash_source_url = ctx_properties['logstash_rpm_source_url']
-    postgresql_jdbc_driver_url = (
-        'https://jdbc.postgresql.org/download/postgresql-9.4.1212.jar'
-    )
-    logstash_output_jdbc_url = (
+def install_logstash_output_jdbc_plugin():
+    """"Install output plugin needed to write to SQL databases."""
+    plugin_url = (
         'https://rubygems.org/downloads/logstash-output-jdbc-0.2.10.gem'
     )
 
-    logstash_log_path = '/var/log/cloudify/logstash'
-
-    ctx.logger.info('Installing Logstash...')
-    utils.set_selinux_permissive()
-    utils.copy_notice(LOGSTASH_SERVICE_NAME)
-
-    utils.yum_install(logstash_source_url, service_name=LOGSTASH_SERVICE_NAME)
-
     ctx.logger.info('Installing logstash-output-jdbc plugin...')
-    logstash_output_jdbc_path = join(
-        tempfile.gettempdir(),
-        basename(logstash_output_jdbc_url),
-    )
-    utils.download_file(
-        logstash_output_jdbc_url,
-        logstash_output_jdbc_path,
-    )
+    plugin_path = join(tempfile.gettempdir(), basename(plugin_url))
+    utils.download_file(plugin_url, plugin_path)
     utils.run([
         'sudo', '-u', 'logstash',
-        '/opt/logstash/bin/plugin', 'install', logstash_output_jdbc_path,
+        '/opt/logstash/bin/plugin', 'install', plugin_path,
     ])
 
+
+def install_postgresql_jdbc_driver():
+    """Install driver used by the jdbc plugin to write data to postgresql."""
+    driver_url = (
+        'https://jdbc.postgresql.org/download/postgresql-9.4.1212.jar'
```

Comment:
Yes, you're right. And the same thing happens with the logstash output plugin. Let me fix that.